.github/workflows/CI.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
I'd suggest limiting the execution of CI to the `devel` branch (and to PRs that target devel) which will reduce the number of parallel executions that can occur when you push to a branch and then create a PR. Additionally helps reduce the load on the limited number of Actions Runners we all share in the LLNL org.

https://github.com/LLNL/Spindle/blob/d5e582f316e03f23afcf6f410b06566ebade5f3c/.github/workflows/ci.yml#L2-L8

Reply:
I will do this, thanks